### PR TITLE
Pbw 4515

### PR DESCRIPTION
**Pull request by chibzu**

### CODE REVIEW

File: skin.json [hunk 1]
```diff
@@ -196,7 +196,7 @@
   "icons": {
     "play": {"fontFamilyName": "alice", "fontString": "\u0068", "fontStyleClass": "icon icon-play"},
     "pause": {"fontFamilyName": "alice", "fontString": "\u0067", "fontStyleClass": "icon icon-pause"},
-    "volume": {"fontFamilyName": "alice", "fontString": "\u0062", "fontStyleClass": "icon icon-volume-desktop"},
+    "volume": {"fontFamilyName": "alice", "fontString": "\u0061", "fontStyleClass": "icon icon-volume-desktop"},
```
Comment (lental) [inline skin.json]: this is legit?  changing the icon over one? what was it before?

File: skin.json [hunk 2]
```diff
@@ -214,6 +214,7 @@
     "right": {"fontFamilyName": "alice", "fontString": "\u0073", "fontStyleClass": "icon icon-right"},
     "learn": {"fontFamilyName": "alice", "fontString": "\u0074", "fontStyleClass": "icon icon-learn"},
     "skip": {"fontFamilyName": "alice", "fontString": "\u0075", "fontStyleClass": "icon icon-skip"},
-    "warning": {"fontFamilyName": "fontawesome", "fontString": "\uf06a", "fontStyleClass": ""}
+    "warning": {"fontFamilyName": "fontawesome", "fontString": "\uf06a", "fontStyleClass": ""},
+    "auto": {"fontFamilyName": "alice", "fontString": "\u0064", "fontStyleClass": "icon icon-auto"}
```
Comment (lental) [inline skin.json]: what is auto?

Comment (lental) [inline skin.json]: oh, is this.. bitrate auto?

File: skin.json (hunk 1 above)
Comment (chibzu) [inline skin.json]: 61 is the 'volume-desktop' icon with one sound line, 62 is the 'volume-mobile' icon with 2 sound lines. We were using 61 in the CSS and just had it misrepresented in the skin.json

File: skin.json (hunk 2 above)
Comment (chibzu) [inline skin.json]: Yes, this is the icon for automatic bitrate in the selection dialog, the box with an 'A' in it.

**Pull request merged**